src/resultsdisassemblypage.ui

```diff
@@ -34,13 +34,76 @@
     </widget>
    </item>
    <item>
-    <widget class="KSqueezedTextLabel" name="filenameLabel">
-     <property name="text">
-      <string>KSqueezedTextLabel</string>
-     </property>
-     <property name="textElideMode">
-      <enum>Qt::ElideRight</enum>
-     </property>
+    <widget class="QWidget" name="widget_4" native="true">
```

Comment:
can you please adjust the search bar to share its horizontal layout with the customSourceCodeHighlighting widget? This would "save" a UI line and would then also be reasonable to be always visible.

Reply:
I agree that this isn't better; but isn't there a horizontal "container" layout that can take two widgets side by side?

Reply:
There is QHorizontalLayout but it still looks weird

![grafik](https://user-images.githubusercontent.com/82457690/211544754-7dafc023-c1d8-4105-916a-9e11651f3d6f.png)

Reply:
by using three widgets it does no longer align with the two code views